src/Adapters/Auth/apple.js

```diff
@@ -64,8 +86,16 @@ const verifyIdToken = async ({ token, id }, clientID) => {
 };
 
 // Returns a promise that fulfills if this id token is valid
-function validateAuthData(authData, options = {}) {
-  return verifyIdToken(authData, options.client_id);
+function validateAuthData(
+  authData,
+  options = { cacheMaxEntries: 5, cacheMaxAge: ONE_HOUR_IN_MS }
+) {
+  return verifyIdToken(
+    authData,
+    options.client_id,
+    options.cacheMaxEntries,
+    options.cacheMaxAge
+  );
```

Comment:
A quick nit. Functions should have 2 parameters max. Just pass in options.

Also if I pass in { clientID: “something” } you default values would never be set. Use the or operator.

options.cacheMaxEntries || 5 ( if the first expression is false or undefined use the second) just a quick trick for setting defaults.

Reply:
Good feedback thanks.  I got that changed.